mlir/test/Dialect/Ptr/layout.mlir

```diff
@@ -0,0 +1,87 @@
+// RUN: mlir-opt --test-data-layout-query --split-input-file --verify-diagnostics %s | FileCheck %s
+
+module attributes { dlti.dl_spec = #dlti.dl_spec<
+  #dlti.dl_entry<!ptr.ptr, dense<[32, 32, 64]> : vector<3xi64>>,
```

Comment:
This is a bit obscure and fragile to me, what about a dedicated attribute?
```
#dlti.dl_entry<!ptr.ptr, #ptr.spec<size=32, abi=32, preferred=64>>,
```

Reply:
I agree, I'll add the attribute.